tuna/session_mixin.py

```diff
@@ -82,7 +88,7 @@ def insert_session(self):
         LOGGER.warning("Err occurred trying to add new session: %s \n %s", err,
                        self)
         session.rollback()
-        entry = self.get_query(session, type(self), self).one()
+        entry = self.get_query(session, type(self), self).one()  #type: ignore
```

Comment:
why is this ignored? this should be of type SessionExample or Session.

Reply:
overload function defiition added